paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerAdapter.kt

```diff
@@ -12,6 +13,8 @@ import com.stripe.android.model.PaymentMethod
  *
  * Implement this interface if you would prefer retrieving and updating your Stripe customer object
  * via your own backend instead of using [StripeCustomerAdapter].
+ *
+ * Use [CustomerAdapter.create] to create an instance of [StripeCustomerAdapter].
```

Comment:
Merchants won't know about `StripeCustomerAdapter` it'll be totally internal! I think we should remove references to it entirely.